src/components/Videos.vue

```diff
@@ -121,6 +127,232 @@ export default {
+					lb: 'Luxembourgish',
+					lg: 'Ganda',
+					li: 'Limburgan',
+					ln: 'Lingala',
+					lo: 'Lao',
+					lt: 'Lithuanian',
+					lu: 'Luba-Katanga',
+					lv: 'Latvian',
+					mg: 'Malagasy',
+					mh: 'Marshallese',
+					mi: 'Maori',
+					mk: 'Macedonian',
+					ml: 'Malayalam',
+					mn: 'Mongolian',
+					mr: 'Marathi',
+					ms: 'Malay',
+					mt: 'Maltese',
+					my: 'Burmese',
+					na: 'Nauru',
+					nb: 'Norwegian Bokmål',
+					nd: 'Ndebele, North',
+					ne: 'Nepali',
+					ng: 'Ndonga',
+					nl: 'Dutch',
+					nn: 'Norwegian Nynorsk',
+					no: 'Norwegian',
+					nr: 'Ndebele, South',
+					nv: 'Navajo',
+					ny: 'Nyanja',
+					oc: 'Occitan',
+					oj: 'Ojibwa',
+					om: 'Oromo',
+					or: 'Oriya',
+					os: 'Ossetian',
+					pa: 'Panjabi',
+					pi: 'Pali',
+					pl: 'Polish',
+					ps: 'Pushto',
+					pt: 'Portuguese',
+					qu: 'Quechua',
+					rm: 'Romansh',
+					rn: 'Rundi',
+					ro: 'Romanian',
+					ru: 'Russian',
+					rw: 'Kinyarwanda',
+					sa: 'Sanskrit',
+					sc: 'Sardinian',
+					sd: 'Sindhi',
+					se: 'Northern Sami',
+					sg: 'Sango',
+					si: 'Sinhala',
+					sk: 'Slovak',
+					sl: 'Slovenian',
+					sm: 'Samoan',
+					sn: 'Shona',
+					so: 'Somali',
+					sq: 'Albanian',
+					sr: 'Serbian',
+					ss: 'Swati',
+					st: 'Sotho',
+					su: 'Sundanese',
+					sv: 'Swedish',
+					sw: 'Swahili',
+					ta: 'Tamil',
+					te: 'Telugu',
+					tg: 'Tajik',
+					th: 'Thai',
+					ti: 'Tigrinya',
+					tk: 'Turkmen',
+					tl: 'Tagalog',
+					tn: 'Tswana',
+					to: 'Tonga',
+					tr: 'Turkish',
+					ts: 'Tsonga',
+					tt: 'Tatar',
+					tw: 'Twi',
+					ty: 'Tahitian',
+					ug: 'Uighur',
+					uk: 'Ukrainian',
+					ur: 'Urdu',
+					uz: 'Uzbek',
+					ve: 'Venda',
+					vi: 'Vietnamese',
+					vo: 'Volapük',
+					wa: 'Walloon',
+					wo: 'Wolof',
+					xh: 'Xhosa',
+					yi: 'Yiddish',
+					yo: 'Yoruba',
+					za: 'Zhuang',
+					zh: 'Chinese',
+					zu: 'Zulu',
+				}
+				const davDir = this.davPath.replace(/[^/]*$/, '')
+				const videoRoot = this.basename.replace(/[.][^.]+$/, '')
+				// Create caption tracks for the HTML5 player
+				// E.g.: <file>.mkv: look for <file>.xx.vtt or .<file>.xx.vtt
+				let capTracks = '' // '<source src="' + this.davPath + '" />\n'
+				for (let i = 0; i < folder.length; ++i) {
+					const basename = folder[i].basename
+					const index = basename.indexOf(videoRoot)
+					// Consider only file... or .file...
+					if (!(index === 0 || (index === 1 || basename[0] === '.'))) {
+						continue
+					}
+					const suffix = basename.slice(videoRoot.length + index)
+					// Consider only ...xx.vtt
+					if (suffix.search(/^[.]..[.]vtt$/) !== 0) {
+						continue
+					}
+					const lang = suffix.slice(1, 3)
+					const language = languages[lang] || lang
+					capTracks += '<track src="' + davDir + basename + '"'
+						+ ' label="' + language + '"'
+						+ ' kind="captions"'
+						+ ' srclang="' + lang + '"'
+						+ ' />\n'
+				}
+				this.tracks = capTracks
```

Comment:
This line should return a list of tracks as specified in the first comment.

Reply:
Done :)